art/paths.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+CHECKPOINTS_PATH = Path("checkpoints")
+LOG_PATH = Path("logs")
+EXPERIMENT_LOG_PATH = CHECKPOINTS_PATH/"experiment"/"logs"
+
+
+def get_checkpoint_step_dir_path(step_id: str, step_name: str) -> Path:
+    """
```

Comment:
I like this is taken out from jsonsavers

Reply:
ok